site/config.toml

```
@@ -20,6 +20,7 @@ theme = "nightfall"
 author = "rust-bitcoin"
 user = "dev"
 hostname = "rust-bitcoin.org"
+footerText = "This project is licensed under the Creative Commons Zero v1.0 Universal License."
```

I'd prefer to keep the mention of Hugo and link to the original theme to give attribution to them, it was totally their work that makes the site look how it does.

```suggestion
footerText = "License CC0-1.0. Built with <a href="https://gohugo.io" class="footerLink">Hugo</a> and <a href="https://github.com/LordMathis/hugo-theme-nightfall" class="footerLink">Nightfall</a> theme"
```

---

I did not test that the HTML embedded in the `footerText`  string works.

---

I tested it and it actually just picks up the strings without any HTML parsing. We need a whole separate script just to parse it. I would suggest instead to Allow the footerText to be limited to only `License CC0-1.0.` and the rest hugo and nightfall links I can hardcode it in my fork. This is the real implementation which nightfall already was using with the difference being that It takes the author name instead of what here we are taking footer-name and rest is hardcoded.

refer 
https://github.com/LordMathis/hugo-theme-nightfall/blob/992c9fc416839e1b896845884427dc85e4cb9fce/layouts/partials/footer.html#LL2C8-L2C8

---

Sounds good to me :)